app/views/studies/_shared_sync_functions.js.erb

```diff
@@ -1,7 +1,8 @@
 // shared JS functions for sync forms
 
 // Used for sync CSFV
-window.SCP.readOnlyToken = '<%= get_read_access_token(@study, current_user) %>'
+window.SCP.readOnlyTokenObject = <%= raw get_read_access_token(@study, current_user).to_json %>
```

Comment:
Good call to catch this!  Wouldn't have known to update this here but I guess that's what global find/replace is for...